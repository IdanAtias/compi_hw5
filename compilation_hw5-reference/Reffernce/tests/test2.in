print "#1) 'I'll try!' she said as he walked away,";
	{
		int donnyJep = 0;
		int kimJerry = 1;
		if (donnyJep == kimJerry)
			print "Error #1";
		if (donnyJep != kimJerry)
			print "#2) 'try not to lose you.'";
		if (donnyJep <= kimJerry)
			print "#3) Two vibrant hearts could change.";
		if (donnyJep >= kimJerry)
			print "Error #2";
	}