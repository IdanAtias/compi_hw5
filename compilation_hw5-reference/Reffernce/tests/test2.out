#1) 'I'll try!' she said as he walked away,
#2) 'try not to lose you.'
#3) Two vibrant hearts could change.
